src/player.cpp

```diff
@@ -536,6 +536,20 @@ PlayerErrors Player::play(
         return bufferStreamCanBePlayedOnlyOnce;
     }
 
+    // Check if by playing this sound will exceed the maximum number of voice count. If true, then
+    // check if [soudHash] has other instances playing. If true remove the first and play the new one.
+    // If there are no other instances playing, this sound cannot be played and return an error.
+    // Issue https://github.com/alnitak/flutter_soloud/issues/204
+    if (getActiveVoiceCount() >= getMaxActiveVoiceCount())
```

Comment:
Just checking: is either of these functions "expensive"? Like, is it just checking a number, or is it counting an array length? Or, worse, trying to walk some more complicated collection?